src/main/java/seedu/address/model/person/Priority.java

```diff
@@ -12,18 +12,18 @@
 public class Priority {
 
     // Regex to validate the priority value (LOW, MEDIUM, or HIGH).
-    public static final String VALIDATION_REGEX = "LOW|MEDIUM|HIGH";
+    public static final String VALIDATION_REGEX = "NONE|LOW|MEDIUM|HIGH";
 
     // Error message for invalid priority values.
-    public static final String MESSAGE_CONSTRAINTS = "Priority should only be 'LOW', 'MEDIUM', or 'HIGH'.";
+    public static final String MESSAGE_CONSTRAINTS = "Priority should only be 'NONE', 'LOW', 'MEDIUM', or 'HIGH'.";
 
     private final String priority;
```

Comment:
Actually I wonder if using Enum would have been easier, then to get the string use `Enum.valueOf(x)` and stuff. But for now it looks good to me.